### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by yannj-fr** [jira-node/node-jira-client#232]
I am wondering if this project is still active, I see PRs not merged for a while and issues with no answer.

**Comment by timo-xo** [comment]
@mtscout6 ^

**Comment by dkokic** [comment]
I would be happy to help if the owners are occupied with other projects.

**Comment by dkokic** [comment]
@mtscout6 I agree. I must admit I never setup a CD pipe on Github, but hopefully it is not a rocket science. My profile on npm is [kodra](https://www.npmjs.com/~kodra)

**Comment by gedrick** [comment]
I wish I could help more with this project but I don't require the level of JIRA use for work anymore, so I haven't used it in quite a while! Huge thanks to the volunteers in this thread though.
